build.gradle

```diff
@@ -833,6 +837,32 @@ project('spring-xd-module-spi') {
 	}
 }
 
+project('spring-xd-spark-streaming') {
+	description = 'Spring XD spark streaming module'
+	dependencies {
+		compile("org.apache.spark:spark-streaming_$sparkScalaVersion:$sparkVersion") {
+			exclude group: 'hsqldb'
+		}
+		compile "org.springframework.integration:spring-integration-core"
+		compile project(":spring-xd-module-spi")
+	}
+}
+
+project('spring-xd-spark-streaming-tests') {
```

Comment:
Not saying that this is bad, but definitely not the approach we have for testing other modules, right?